docs/resources/integration.md

```diff
@@ -116,6 +117,14 @@ Required:
 - `zone` (String) Grafana Zone.
 
 
+<a id="nestedatt--prometheus_spec"></a>
+### Nested Schema for `prometheus_spec`
+
+Required:
+
+- `endpoint` (String) Prometheus OTLP endpoint URL
```

Comment:
`Prometheus OTLP endpoint URL of the form http://my-prometheus-endpoint/api/v1/otlp`
Perhaps?

Reply:
fixed.